### PR TITLE
Initialise the DB

### PR DESCRIPTION
**Pull request by mafonso**
### Context

Run DB Migrations on startup

### CODE REVIEW

File: startup.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+set -e
+
+bundle exec rake db:create
```
Comment (smoothcontract) [inline startup.sh]: I don't think this should run `db:create` every time the container starts - assumption would be that we create an initial empty database as a one-off activity.

Comment (mafonso) [reply startup.sh]: I checked with @tavy88 that is was idempotent.  Trying to avoid one-off tasks as much as possible.

Comment (smoothcontract) [reply startup.sh]: Okay then, in that case I'm happy.

Comment (cwrw) [reply startup.sh]: I agree with @smoothcontract we have to be very explicit on `db:create` and ideally should not include that in the startup script

**Pull request merged**